make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -200,3 +200,7 @@ ALTER TABLE replication_task ALTER COLUMN dst_resource TYPE varchar(512);
 /*remove count from quota hard and quota_usage used json*/
 UPDATE quota SET hard = hard - 'count';
 UPDATE quota_usage SET used = used - 'count';
+
+/* make Clair and Trivy as reserved name for scanners in-tree */
+UPDATE scanner_registration SET name = concat_ws('-', name, uuid) WHERE name IN ('Clair', 'Trivy') AND immutable = FALSE;
```

Comment:
we do need to document it as the name will be changed if user is using the external clair or trivy.

Reply:
This will be in docs, maybe also in release note.

Reply:
@heww 

release notes may be more proper.